server/views/serviceProviderReferrals/referralDetails.njk

```diff
@@ -1,23 +1,8 @@
-{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
-{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
-{% from "govuk/components/input/macro.njk" import govukInput %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
-
-{% extends "../partials/layout.njk" %}
-
-{% set pageTitle = "HMPPS Interventions" %}
-{% block pageTitle %}
-  {{ pageTitle }}
-  - GOV.UK
-{% endblock %}
-
-{% block content %}
-  {{ govukNotificationBanner(serviceUserNotificationBannerArgs) }}
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">{{ presenter.text.title }}</h1>
```

Comment:
Is this presenter property no longer used? Can we get rid of it if so?

Reply:
done